app/view_objects/cloud_cover_url.rb

```diff
@@ -0,0 +1,31 @@
+class CloudCoverUrl
+  include CloudinaryHelper
+  include ActionView::Helpers::AssetUrlHelper
+
+  def initialize(url)
+    @url = url
+  end
+
+  def call
+    return if url.blank?
+    return url if Rails.env.development? || Rails.env.test?
```

Comment:
We used to have `return asset_path("triple-unicorn") if Rails.env.test?` here.
After moving the code to the view object I started getting a lot of `ActiveRecord::NotFound` errors in `system` tests because there is no  `/triple-unicorn` path. Maybe I misunderstood something and triple-unicorn :unicorn: :sparkles:  is essential, then, please, correct me.

Reply:
I'm not 100% sure how these things came to be, but "triple-unicorn" was a specific asset from the past but is no longer an app concept, but I guess it was picked for this random use.

Anyway, yeah you are correct.